Themes/default/css/index.css

```diff
@@ -1431,8 +1433,10 @@ ul li.greeting {
 	background-position: 1px -210px;
 	padding: 0 4px 0 22px;
 }
-.quickbuttons li a.unapprove_button {
+.quickbuttons li a.unapprove_button,
+.quickbuttons li a.unread_button {
 	background-position: 2px -238px;
+	padding: 0 4px 0 22px;
```

Comment:
Knocks bottom button out of alignment IMO.
![fireshot screen capture 169 - automated test 632 - localhost_smf2_1_index_php_topic 635_msg30071 new](https://cloud.githubusercontent.com/assets/431799/6079864/0181b85e-adc8-11e4-8808-f006a44a7ed5.png)